pageserver/src/walredo.rs

```diff
@@ -1010,3 +1010,112 @@ fn build_get_page_msg(tag: BufferTag, buf: &mut Vec<u8>) {
     tag.ser_into(buf)
         .expect("serialize BufferTag should always succeed");
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{PostgresRedoManager, WalRedoManager};
+    use crate::repository::Key;
+    use crate::{config::PageServerConf, walrecord::NeonWalRecord};
+    use bytes::Bytes;
+    use std::str::FromStr;
+    use utils::{id::TenantId, lsn::Lsn};
+
+    #[test]
+    fn short_v14_redo() {
+        // prettier than embedding the 8192 bytes here though most of it are zeroes
+        // PRE-MERGE: zstd would cut it to 263 bytes, consider in review?
+        let expected = std::fs::read("fixtures/short_v14_redo.page").unwrap();
```

Comment:
I have suggested around zstd usage, but we have already selected the path to add large fixtures as part of `main`, so I guess this 8192 bytes file is ok? Will cleanup these comments after a review round.

Reply:
IMO if zstd usage requires compiling zstd stuff as part of the test suite it is a downside. To me 8k file looks ok